app/views/users/edit_security.html.erb

```diff
@@ -92,6 +92,36 @@
       Sign in to HCB using your fingerprint or a hardware security key.
     </div>
   <% end %>
+  <div class="flex items-center justify-between flex-wrap gap-x-5 gap-y-2 mt3 pb2" id="backup-codes">
+    <legend class="flex flex-row justify-start items-center">
+      <span class="h2 bold">Backup codes</span>
+    </legend>
+    <div class="flex flex-row gap-x-2">
+      <% if @user.backup_codes_enabled? %>
+        <%= link_to user_disable_backup_codes_path(@user), class: "btn px2 h4 bg-error", method: :post, data: { turbo: false, confirm: "Are you sure? This will invalidate your currently active codes immediately." } do %>
```

Comment:
```suggestion
        <%= link_to user_disable_backup_codes_path(@user), class: "btn px2 h4 bg-error", method: :post, data: { turbo: false, confirm: "Are you sure? This will invalidate your current codes immediately." } do %>
```